multi-node/aws/pkg/config/config_test.go

```diff
@@ -34,6 +34,11 @@ vpcId: vpc-xxxxx
 routeTableId: rtb-xxxxxx
 `, `
 vpcId: vpc-xxxxx
+`, `
+createRecordSet: false
+hostedZone: ""
+`, `
+recordSetTTL: 400
```

Comment:
hmm yeah, there's no good reason to ever have that combo, i'll make it invalid and improve the tests